universal-login-commons/test/core/utils/deepMerge.ts

```diff
@@ -0,0 +1,35 @@
+import {expect} from 'chai';
+import {deepMerge} from '../../../lib/core/utils/deepMerge';
+
+describe('UNIT: deepMerge', () => {
+  const obj1 = {
+    a1: 1,
+    a2: 2,
+    b: {
+      c1: 1,
+      c2: 2,
+    }
+  };
+
+  const obj2 = {
+    a1: 10,
+    b: {
+      c2: 20,
+    }
+  };
+
```

Comment:
it('empty merged to non-empty')
it('non-empty merged to empty')
it('flat merged to flat')
it('3 levels')